detect_secrets/plugins/jwt.py

```diff
@@ -2,8 +2,11 @@
 This plugin finds JWT tokens
 """
 import base64
+import jwt
 import json
 import re
+import datetime  
```

Comment:
This line is not needed since you redeclare the `datetime` variable in the line right after this.